### ISSUE TITLE
Allow dynamic imports for blob and data urls

### ISSUE THREAD

**Issue by saviski** [Jack-Works/ttypescript-browser-like-import-transformer#18]
I see that there are 2 functions that check for http(s)
I would like to be able to bypass the url rewrite for blob and data urls when using dynamic imports.

```javascript
let url = "blob:http://site.com/hash";
let url2 = "data:text/javascript,...";
import(url);
import(url2);
```

now it converts it to 

`/web_modules/blob:http://...`

```javascript

    function isBrowserCompatibleModuleSpecifier(path) {
        return isHTTPModuleSpecifier(path) || isLocalModuleSpecifier(path);
    }
    function isHTTPModuleSpecifier(path) {
        return path.startsWith('http://') || path.startsWith('https://');
    }
```

I guess it is just a matter of checking for data: and blob: on those functions

**Comment by Jack-Works** [comment]
Oh sure! Thanks for your support! I'll fix it soon!

**Comment by Jack-Works** [comment]
@saviski hi, you can update to 2.0.3, I have fixed this

**Comment by saviski** [comment]
Thanks for the fast reply.
I just had to rebuild it to generate a new `es/ttsclib.js`


https://cdn.jsdelivr.net/npm/@magic-works/ttypescript-browser-like-import-transformer@2.0.3/es/ttsclib.min.js
looks outdated, it is missing **isDataOrBlobModuleSpecifier**

**Comment by Jack-Works** [comment]
Thanks for point the mistake! I have released 2.0.4 to fix that!
@saviski
